api/ReadableStream.json

```diff
@@ -418,6 +418,74 @@
             "deprecated": false
           }
         }
+      },
+      "values": {
+        "__compat": {
+          "spec_url": "https://streams.spec.whatwg.org/#readablestream",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "110"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "@@asyncIterator": {
```

Comment:
This is good, but:
1. YOu should delete the "effective duplicate" key on line 95.
2. This is supported by node, as per the key on line 95